internal/gcsx/integration_test.go

```diff
@@ -137,11 +136,6 @@ func (t *IntegrationTest) objectGeneration(name string) (gen int64) {
 		panic(err)
 	}
 
-	// Check the result.
-	if o.Generation > math.MaxInt64 {
-		panic(fmt.Sprintf("Out of range: %v", o.Generation))
```

Comment:
why is this removed?

Reply:
This is an pointless check, it's comparing an int64 value `>` MaxInt64, which is always false by definition.